Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN npm install
 COPY ./ .
 RUN npm run build
 
-FROM nginx:1.18.0 as production-stage
+# FROM nginx:1.18.0 as production-stage
```

Comment:
Patrick says we might use Dockerfile in the future (same as back end) so we should keep this file. I commented this line out as we don't use nginx any more.

Reply:
I reverted this change. Getting rid of nginx is a problem for another day. Resolving.

cc @pwei1018